pkg/apis/devops/v1alpha1/s2irun_types.go

```diff
@@ -61,8 +61,50 @@ type S2iRunStatus struct {
 	LogURL string `json:"logURL,omitempty"`
 	//KubernetesJobName is the job name in k8s
 	KubernetesJobName string `json:"kubernetesJobName,omitempty"`
+
+	// S2i build result info.
+	S2iBuildResult *S2iBuildResult `json:"s2iBuildResult,omitempty"`
+	// S2i build source info.
+	S2iBuildSource *S2iBuildSource `json:"s2iBuildSource,omitempty"`
+}
+
+type S2iBuildResult struct {
 	//ImageName is the name of artifact
 	ImageName string `json:"imageName,omitempty"`
+	//The size in bytes of the image
+	ImageSize int64 `json:"imageSize,omitempty"`
+	// Image ID.
+	ImageID string `json:"imageID,omitempty"`
+	// Created time.
+	ImageCreated string `json:"imageCreated,omitempty"`
```

Comment:
createTime ? use meta.Time?

Reply:
ImageCreated and meta.Time are similar, but describe different things. ImageCreated is the time of image created, get from docker inspect api.
Maybe the comment here is not clear.